resource-service/README.md

```diff
@@ -116,7 +116,7 @@ curl -X POST api/resource -H "content-type: application/json" --data \
          
       },
       "connectionDetail":{
-         "class":"uk.gov.gchq.palisade.service.SimpleConnectionDetail",
+         "class":"uk.gov.gchq.palisade.resource.impl.SimpleConnectionDetail",
```

Comment:
is the windows related part of this readme still relevant? line 141

Reply:
I am hesitant to change this as it should be dealt with as part of the resource service documentation ticket which is still open.

Reply:
although true, this ticket moved the hadoop configuration stuff to readers so documentation should probably be moved too

Reply:
Okay, this section has been removed.